docs/getting-started/compose-infrastructure.md

```diff
@@ -66,10 +64,8 @@ spec:
     group: database.example.org
     version: v1alpha1
     names:
-      kind: PostgreSQLInstance
-      listKind: PostgreSQLInstanceList
-      plural: postgresqlinstances
-      singular: postgresqlinstance
+      kind: CompositePostgreSQLInstance
```

Comment:
I like how we call the definition `Composite____` and then the claim the actual thing here :+1: